lints/lint_sub_cert_valid_time_longer_than_39_months.go

```diff
@@ -0,0 +1,36 @@
+// lint_sub_cert_valid_time_longer_than_39_months.go
```

Comment:
ugh why do we still insist on putting useless comments at the top of files? I thought I went through and deleted all of these. @justinbastress, not your fault for matching to what have, but we should either actually document the lint here or just omit the comment.

Reply:
I agree with @dadrian, this seems like something that can (and probably should!) go.

Reply:
I can put together a quick script to strip these and push a new PR with the results.